source/ref_installation.md

```diff
@@ -0,0 +1,415 @@
+    client_email: service-account@my-gcp-project-id-zzzzz.iam.gserviceaccount.com
+    client_id: 000000000000000000000
+    auth_uri: https://accounts.google.com/o/oauth2/auth
+    token_uri: https://oauth2.googleapis.com/token
+    auth_provider_x509_cert_url: https://www.googleapis.com/oauth2/v1/certs
+    client_x509_cert_url: https://www.googleapis.com/robot/v1/metadata/x509/service-account%40my-gcp-project-id-zzzzz.iam.gserviceaccount.com
+EOF
+
+$ helm install odahu/odahu-flow-fluentd --name fluentd --namespace fluentd --values ./fluentd_values.yaml
+```
+
+### 2.8. Install [Prometheus operator](https://github.com/coreos/prometheus-operator) (optional)
+
+TBD.
+
+## 3. Install ODAHU
+
+### 3.1. Install ODAHU services
+
+Create namespace for core ODAHU service:
+```bash
+$ kubectl create namespace odahu-flow &&\
+  kubectl label namespace odahu-flow project=odahu-flow
+```
+
+Create namespace for ODAHU deployment and label it for Istio injection:
+```bash
+$ kubectl create namespace odahu-flow-deployment &&\
+  kubectl label namespace odahu-flow-deployment project=odahu-flow istio-injection=enabled
+```
+
+Prepare YAML config with values for `odahu-flow` Helm chart.
+
+Example:
+```bash
+$  cat << EOF > odahuflow_values.yaml
+odahuFlowVersion: 1.0.0
+logLevel: debug
+rbac: true
+edge:
+  knative_namespace: knative-serving
+  ingress:
+    enabled: false
+ingress:
+  enabled: false
+security:
+  enabled: false
+metrics:
+  enabled: false
+feedback:
+  enabled: true
+  istio_namespace: istio-system
+  rq_catcher:
+    prohibited_headers:
+    - x-secret-header
+  collector:
+    replicas: 1
+  fluentd:
+    host: fluentd.fluentd.svc.cluster.local
+    port: 24224
+operator:
+  server:
+    replicas: 1
+api:
+  enabled: true
+  ingress:
+    enabled: false
+  port: 80
+  replicas: 2
+service_catalog:
+  enabled: false
+config:
+  common:
+    external_urls:
+    - name: Documentation
+      url: https://docs.odahu.org
+  operator:
+    api_url: http://odahu-flow-api.odahu-flow.svc.cluster.local
+  connection:
+    enabled: true
+    repository_type: kubernetes
+  deployment:
+    enabled: true
+    namespace: odahu-flow-deployment
+  packaging:
+    enabled: true
+    namespace: odahu-flow
+    output_connection: models-output
+    service_account: odahu-flow-model-packager
+  training:
+    enabled: true
+    namespace: odahu-flow
+    output_connection: models-output
+connections:
+- id: odahu-flow-examples
+  spec:
+    description: Private git repository with odahu-flow examples
+    keySecret: LS0tLS1CRUdJTiBPUEVOU1NIIFBSSVZBVEUgS0VZLS0tLS0KVGhlcmUgaXMgc29tZSBwcml2YXRlIE9wZW5TU0gga2V5IGZvciBHaXQgcmVwbwotLS0tLUVORCBPUEVOU1NIIFBSSVZBVEUgS0VZLS0tLS0=
+    reference: 1.0.0
+    type: git
+    uri: git@git.example.com:odahu/odahu-examples.git
+    webUILink: https://git.example.com/odahu/odahu-examples
+- id: docker-ci
+  spec:
+    description: Docker registry (GCR) for model packaging
```

Comment:
Specify a link to the docker connection documentation. Maybe we will use the dockerhub as example?

Reply:
Done in 2ecab1c386ed88134fae670b0d8115b72e86ea9d.